BodoSQL/bodosql/imported_java_classes.py

```diff
@@ -49,6 +49,10 @@
         # Initialize logging. Must be done after importing all classes to ensure
         # PythonLoggersClass is available.
         configure_java_logging(bodo.user_logging.get_verbose_level())
+        # Note: We call this JavaEntryPoint so its clear the Python code enters java
+        # and the class is named PythonEntryPoint to make it clear the Java code
+        # is being entered from Python.
+        JavaEntryPoint = gateway.jvm.com.bodosql.calcite.application.PythonEntryPoint
```

Comment:
My hope is that a lot of these other entry points can be removed soon, at least for things like constructors that aren't tested/used in Java.